www/data/docs_sidebar.yml

```diff
@@ -1,122 +1,195 @@
-    - title: Configuration updates
-      link: "/docs/run-packages-apply-config-updates/"
-    - title: Upload files
-      link: "/docs/run-packages-upload-files/"
-    - title: Security
-      link: "/docs/run-packages-security/"
-    - title: Binding
-      link: "/docs/run-packages-binding/"
-    - title: Update strategy
-      link: "/docs/run-packages-update-strategy/"
-    - title: Multiple Services
-      link: "/docs/run-packages-multiple-services/"
-    - title: Export packages
-      link: "/docs/run-packages-export/"
-    - title: Monitoring
-      link: "/docs/run-packages-monitoring/"
-  - title: Share packages
-    link: "/docs/share-packages-overview/"
-  - title: Continuous deployment
-    link: "/docs/continuous-deployment-overview/"
-  - title: Container orchestration
-    link: "/docs/container-orchestration/"
-    sub_links:
-    - title: Amazon EC2 Container Service
-      link: "/docs/container-orchestration-ecs/"
-    - title: Apache Mesos and DC/OS
-      link: "/docs/container-orchestration-mesos/"
-    - title: Kubernetes
-      link: "/docs/container-orchestration-kubernetes/"
-  - title: Habitat internals <span>&plus;</span>
-    link: "/docs/internals-overview/"
+      link: "#glossary-services"
+    - title: Builder
+      link: "#glossary-builder"
+  - title: Internals <span>&plus;</span>
+    link: "/docs/internals"
     sub_links:
     - title: Supervisor
-      link: "/docs/internals-supervisor/"
+      link: "#supervisor-internals"
     - title: Leader Election
-      link: "/docs/internals-leader-election/"
-    - title: Cryptography
-      link: "/docs/internals-crypto/"
-    - title: Bootstrapping Habitat
-      link: "/docs/internals-bootstrapping/"
-- title: Reference
-  links:
-  - title: CLI reference
-    link: "/docs/reference/habitat-cli/"
-  - title: Plan syntax
-    link: "/docs/reference/plan-syntax/"
-    sub_links:
-    - title: Basic settings
-      link: "/docs/reference/basic-settings/"
-    - title: Callbacks (Build Phases)
-      link: "/docs/reference/callbacks/"
-    - title: Build variables
-      link: "/docs/reference/build-variables/"
-    - title: Application Lifecycle Hooks
-      link: "/docs/reference/hooks/"
-    - title: Runtime settings
-      link: "/docs/reference/runtime-settings/"
-    - title: Utility functions
-      link: "/docs/reference/utility-functions/"
-  - title: Environment variables
-    link: "/docs/reference/environment-vars/"
-  - title: Package contents
-    link: "/docs/reference/package-contents/"
-  - title: Supervisor log keys
-    link: "/docs/reference/log-keys/"
-  - title: Habitat infographics
-    link: "/docs/reference/habitat-infographics/"
-- title: Contribute
-  links:
-  - title: Help build Habitat
-    link: "/docs/contribute-help-build/"
+      link: "#election-internals"
+    - title: Crypto
+      link: "#crypto-internals"
+    - title: Bottstrapping Habitat
+      link: "#bootstrap-internals"
+  - title: Contribute
```

Comment:
Is this a duplicate of "Core Contribution"? If so, why not delete the "Core contribution" article and link under "Best practices guides"

Reply:
Meant to be Core Development guide, fixed by new commit.